From e99083a3f4ad5c64eee6c931f4bc6dfcd6188d78 Mon Sep 17 00:00:00 2001
From: PavanKumarIntel <pavanx.kumar.martha@intel.com>
Date: Mon, 29 Jan 2024 12:27:17 +0000
Subject: [PATCH] Increase MaxFWImageSize to avoid transfer failure

While doing Firmware update KCS via SIO, image is not
transferring properly due to exceeding the max Firmware
image size

Tested:
    After increasing the Max Firmware Image Size, image transferred
    successfully and updated

Signed-off-by: PavanKumarIntel <pavanx.kumar.martha@intel.com>
---
 src/firmware-update.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/firmware-update.cpp b/src/firmware-update.cpp
index 93e9078..02ae69a 100644
--- a/src/firmware-update.cpp
+++ b/src/firmware-update.cpp
@@ -167,7 +167,7 @@ constexpr std::size_t operator""_MB(unsigned long long v)
 {
     return 1024u * 1024u * v;
 }
-static constexpr size_t maxFirmwareImageSize = 32_MB;
+static constexpr size_t maxFirmwareImageSize = 33_MB;
 
 static bool localDownloadInProgress(void)
 {
-- 
2.25.1

